authentication-guides/jira/README.md

```diff
@@ -0,0 +1,267 @@
+
+In this tutorial, we will explore how to use Creator Studio to notify approvers in-bot in near-real time when a new approval is pending.
+
+Let's get started!
+
+# Prerequisites
+
+<aside>
+🛠 **What you need before starting**
+
+- The JIRA Automation Feature enabled in JIRA Service Desk
+- Moveworks Creator Studio
+- Moveworks API Key (Provided by your Customer Success Team)
+- Postman or an API Testing Tool
+- A Request Item in JIRA that requires an Approval
+</aside>
+
+# **Prerequisites**
+
+- The JIRA Automation Feature enabled in JIRA Service Desk
+- Moveworks Creator Studio
+- Moveworks API Key (Provided by your Customer Success Team)
+- Postman or an API Testing Tool
+- A Request Item in JIRA that requires an Approval
+
+# What are we building?
+
+The capability for Moveworks to receive approval notification events from JIRA by leveraging JIRA Automations.
+
+## Conversation Design
+
+<aside>
+💡 **Guidance**
+
+- Add a purple chat mock of the use case
+    - Define it in terms of ✨ triggers, 🤲 slots, 🏃‍♂️ actions, and 📚 guidelines
+- Include table with the design pattern
+</aside>
+
+[This purple chat](https://developer.moveworks.com/creator-studio/purple-chat-builder/?workspace=%7B%22title%22%3A%22My+Workspace%22%2C%22mocks%22%3A%5B%7B%22id%22%3A1636%2C%22title%22%3A%22Mock+1%22%2C%22transcript%22%3A%7B%22settings%22%3A%7B%22colorStyle%22%3A%22LIGHT%22%2C%22startTime%22%3A%2211%3A43+AM%22%2C%22defaultPerson%22%3A%22GWEN%22%2C%22editable%22%3Atrue%2C%22botName%22%3A%22%22%2C%22botImageUrl%22%3A%22%22%7D%2C%22messages%22%3A%5B%7B%22from%22%3A%22USER%22%2C%22text%22%3A%22%3Cp%3EHi%2C+can+you+please+reset+my+Azure+MFA%3F%3Cbr%3E%3C%2Fp%3E%22%7D%2C%7B%22from%22%3A%22BOT%22%2C%22text%22%3A%22%3Cp%3EDo+you+want+to+disable+all+your+Azure+MFA+registrations%3F%3Cbr%3E%3C%2Fp%3E%22%2C%22cards%22%3A%5B%7B%22title%22%3A%22%3Cp%3E%3C%2Fp%3E%22%2C%22text%22%3A%22%3Cp%3E%3C%2Fp%3E%22%2C%22buttons%22%3A%5B%7B%22style%22%3A%22PRIMARY%22%2C%22text%22%3A%22Yes%22%7D%2C%7B%22text%22%3A%22No%22%7D%5D%7D%5D%7D%2C%7B%22from%22%3A%22USER%22%2C%22text%22%3A%22%3Cp%3EYes%3Cbr%3E%3C%2Fp%3E%22%7D%2C%7B%22from%22%3A%22ANNOTATION%22%2C%22text%22%3A%22%3Cp%3EInbound+Request+to+Middleware%3A%3Cbr%3E%3Cbr%3E%7B%3Cbr%3E%5C%22email%5C%22%3A+%5C%22gwen%40moveworks.ai%5C%22%3Cbr%3E%7D%3Cbr%3E%3Cbr%3EProcess%3A%3Cbr%3E1.+Establish+a+connection+to+Azure+MFA+Services%3Cbr%3E2.+List+all+MFA+devices+for+the+user%3Cbr%3E3.+Delete+each+MFA+device%2C+this+will+clear+the+user%27s+devices.%3Cbr%3E%3Cbr%3EOutbound+Response%3A%3Cbr%3E%7B+%5C%22status%5C%22%3A+%5C%22ok%5C%22+%7D%2C+200%3Cbr%3E%3C%2Fp%3E%22%7D%2C%7B%22from%22%3A%22BOT%22%2C%22text%22%3A%22%3Cp%3EI+have+successfully+reset+your+MFA%3Cbr%3E%3C%2Fp%3E%22%7D%5D%7D%7D%5D%2C%22botSettings%22%3A%7B%22name%22%3A%22%22%2C%22imageUrl%22%3A%22%22%7D%7D) shows the experience we are going to build.
+
+![purple-chat.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/purple-chat.png)
```

Comment:
Please remove this -- you just need the Purple Chat Link and the docs site will automatically expand it!